Update Instructions for Argo Workflows (#2382)
kubeflow/katib: examples/v1beta1/argo/README.md

  -p='[{"op": "add", "path": "/rules/-", "value": {"apiGroups":["argoproj.io"],"resources":["workflows"],"verbs":["get", "list", "watch", "create", "delete"]}}]'
```

> Run the following command to update katib-config:

Reviewer: ```suggestion
Run the following command to update [Katib config](https://www.kubeflow.org/docs/components/katib/user-guides/katib-config/#katib-controller-parameters):
```

```bash
kubectl edit configMap katib-config -n kubeflow
```

> For example, to support Workflow Pipelines, add `Workflow.v1alpha1.argoproj.io` in `trialResources`:

Reviewer: ```suggestion
For example, to support Argo Workflows, add `Workflow.v1alpha1.argoproj.io` in `trialResources`:
```

```bash
trialResources:
  - Workflow.v1alpha1.argoproj.io
```

> then yo need to delele pod and wait the `katib-controller-xxx` restart

Reviewer: ```suggestion
After that, you need to restart Katib controller Pod:

```bash
kubectl delete pod -n kubeflow -l katib.kubeflow.org/component=controller```
```

```bash

> kubectl delete pod katib-controller-787dc7864f-h627x -n kubeflow

Reviewer: We can remove this.
